README.md

```diff
@@ -20,8 +20,8 @@ Findex is part of Cosmian Cloudproof Encryption.
 - [Getting started](#getting-started)
 - [Building and testing](#building-and-testing)
 - [Findex indexes](#findex-indexes)
-	- [Two indexing strategies](#two-indexing-strategies)
-- [Benchmarks](#benchmarks)
+  * [Two indexing strategies](#two-indexing-strategies)
+  * [Benchmarks](#benchmarks)
```

Comment:
Isn't supposed to be a dedicated section ? I think the problem lies in the level of the actual section title.

Reply:
correct. Wrong section level